src/frontend/packages/suse-extensions/src/custom/kubernetes/workloads/store/workloads-entity-generator.ts

```diff
@@ -7,22 +7,35 @@ import { IFavoriteMetadata } from '../../../../../../store/src/types/user-favori
 import { kubernetesEntityFactory } from '../../kubernetes-entity-factory';
 import { HelmRelease, HelmReleaseGraph, HelmReleaseResources } from '../workload.types';
 import { workloadsEntityCatalog } from '../workloads-entity-catalog';
+import { HelmReleaseHistory } from './../workload.types';
 import {
   WorkloadGraphBuilders,
   workloadGraphBuilders,
   WorkloadReleaseBuilders,
   workloadReleaseBuilders,
   WorkloadResourceBuilders,
   workloadResourceBuilders,
+  WorkloadResourceHistoryBuilders,
+  workloadResourceHistoryBuilders,
 } from './workload-action-builders';
-import { helmReleaseEntityKey, helmReleaseGraphEntityType, helmReleaseResourceEntityType } from './workloads-entity-factory';
+import {
+  helmReleaseEntityKey,
+  helmReleaseGraphEntityType,
+  helmReleaseHistoryEntityType,
+  helmReleaseResourceEntityType,
+  registerWorkloadEntityCatalog,
+} from './workloads-entity-factory';
 
 
 export function generateWorkloadsEntities(endpointDefinition: StratosEndpointExtensionDefinition): StratosBaseCatalogEntity[] {
+
+  registerWorkloadEntityCatalog();
```

Comment:
double check

Reply:
This doesn't seen to be needed, so I have reverted it.